run_meson_command_tests.py

```diff
@@ -31,8 +31,6 @@
     scheme = sysconfig.get_default_scheme()
 else:
     scheme = sysconfig._get_default_scheme()
-if scheme == 'posix_local':
-    scheme = 'posix_prefix'
```

Comment:
This still leaves quite a bit of pointless code floating around here. We should simply do a straight `git revert a2079939de2d1d271e5ea7f23c3656a73e325728` of #10297

Reply:
¨FTR I tested that and things still work.